src/app/components/elements/ReplyEditor.scss

```diff
@@ -87,7 +87,7 @@
 }
 
 .ReplyEditor .Preview {
-  max-width: 50rem;
+  max-width: 42rem;
```

Comment:
Why this drastic reduction of the max-width value? PostFull was at 50rem.

Reply:
Unless you wanted to make it 52rem and remove my margins and replace them with a padding of 1em on either side?

Reply:
OK, gotcha.
42.125rem is actually better as it would render to exactly 640px

Reply:
![Screen Shot 2019-05-03 at 7 34 27 pm](https://user-images.githubusercontent.com/310654/57129752-b9433b80-6dda-11e9-9f38-cfb3ad5b582c.jpg)

Reply:
I opened a new PR